tests/pkcli_data/m1.in/x.py

```diff
@@ -0,0 +1,5 @@
+import pykern.pkcli
```

Comment:
Not sure I understand the naming of this. We had p1, p2, and now m1.in. Perhaps just 1.in? The m1 makes me ask "what does m stand for"?